### ISSUE TITLE
Cannot override SDL2_net with EMCC_LOCAL_PORTS

### ISSUE THREAD

**Issue by fallenoak** [emscripten-core/emscripten#8466]
If I set the `EMCC_LOCAL_PORTS` environment variable to `sdl2_net=/path/to/dir`, I see the following error:

```
system_libs:ERROR: port sdl2_net lacks .SUBDIR attribute, which we need in order to override it locally, please update it
```

Is this because `SUBDIR` is missing in `tools/ports/sdl2_net.py`?

What's necessary to permit the use of `EMCC_LOCAL_PORTS` for a given library?

**Comment by kripken** [comment]
Yeah, we need to define `SUBDIR` in ports for this to work with them. See how the `sdl2.py` does it - should be straightforward to add I think.

**Comment by quantum5** [comment]
A quick inspection showed that only `sdl2.py` has the `SUBDIR` attribute. Perhaps we should think of a better way to implement this feature that doesn't need to set `SUBDIR` on all ports?

**Comment by SkyLeite** [comment]
This is still an issue
